content/en/docs/reference/command-line-tools-reference/feature-gates/kubelet-separate-disk-gc.md

```diff
@@ -6,9 +6,13 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
+  - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    toVersion: "1.30"
  - stage: beta
    defaultValue: true
    fromVersion: "1.31"
```

Reply:
Done.